api/clients/v2/request_signer.go

```diff
@@ -0,0 +1,62 @@
+package clients
```

Comment:
nit: we can update the file name as well?

Reply:
Oops, forgot to rename the file. 😅 Fixed.